fastlane/Fastfile

```diff
@@ -119,3 +119,45 @@ lane :installProfiles do
     FastlaneCore::ProvisioningProfile.install(profile)
   end
 end
+
+desc "Updates project dependencies in Bundler and CocoaPods, then sends a pull request if there are changes. Requires the fastlane-plugin-git_status gem in your Pluginfile."
```

Comment:
Is there a way to check for the presence of the gem as an assertion?

Reply:
Maybe, perhaps using `require`? I’ll investigate.

Reply:
@jshier There’s a `fastlane_require` function you can call that will fail if a gem is not installed. It’s perhaps not as elegant as it could be, and I might send them a PR with something like `fastlane_require_plugin` that targets the error message more towards Fastlane plugins, but this will at least show people that they need to install something. The error looks like this:

```[13:48:52]: fastlane finished with errors

[!] Add 'gem "fastlane-plugin-git_status"' to your Gemfile and restart fastlane```